packages/design-system/src/styles/components/_StepList.scss

```diff
@@ -179,7 +179,7 @@
 
   &::before {
     content: '\2022';
-    left: -$spacer-3;
+    left: -#{$spacer-3};
```

Comment:
Here's another one. Anyone know of a way to turn a positive number into a negative when it's a CSS prop?

Reply:
Yeah I did this a few times, I'll run this through the paces to see how it works

Reply:
Yep have to do `calc(-1 * var(--cssvar))` or `calc(-1 * #{$sass_var})`